tests/ui/traits/issue-77982.rs

```diff
@@ -1,5 +1,4 @@
-//@ ignore-windows different list of satisfying impls
-//@ ignore-arm different list of satisfying impls
```

Comment:
This `//@ ignore-arm` is needed because

```diff
46    = note: multiple `impl`s satisfying `u32: From<_>` found in the `core` crate:
47            - impl From<Char> for u32;
48            - impl From<Ipv4Addr> for u32;
+            - impl From<Simd<u16, 2>> for u32;
+            - impl From<Simd<u8, 4>> for u32;
```